views/article.ejs

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%= title %></title>
+    <link rel="icon" type="image/x-icon" href="/assets/favicon.png" />
+    <link rel="stylesheet" href="css/common.css" />
+    <link rel="stylesheet" href="css/article.css" />
+    <script src="js/navbar.js" defer async></script>
+  </head>
+  <body>
+    <%- include('./components/navbar.ejs') %>
```

Comment:
Let's add a `root` parameter to the navbar because clicking the navigation buttons send us to the `http://localhost:3000/article#about` anchor instead of `http://localhost:3000/#about`

Reply:
Also notice that some blog keys are not required:
<img width="176" alt="Screenshot 2022-11-06 at 3 15 13 PM" src="https://user-images.githubusercontent.com/50573884/200192919-7f8d35b7-d8a1-49cc-98da-d2cadf077731.png">

Let's add an ejs statement that checks if these are not present:
```ejs
<% if (locals.KEY_NAME) { %>
    
<% } %>
```

Reply:
On that note, the blog schema needs a subtitle property, should that be required or optional?

Reply:
It should be optional. Good catch.